src/NzbDrone.Core/Notifications/Telegram/TelegramSettings.cs

```diff
@@ -32,9 +41,25 @@ public class TelegramSettings : IProviderConfig
         [FieldDefinition(3, Label = "NotificationsTelegramSettingsSendSilently", Type = FieldType.Checkbox, HelpText = "NotificationsTelegramSettingsSendSilentlyHelpText")]
         public bool SendSilently { get; set; }
 
+        public bool SendMetadataLink { get; set; }
+        [FieldDefinition(4, Label = "NotificationsTelegramSettingsMetadataLinkType", Type = FieldType.Select, SelectOptions = typeof(MetadataLinkType), HelpText = "NotificationsTelegramSettingsMetadataLinkType")]
+        public MetadataLinkType MetadataLinkType { get; set; }
+
         public NzbDroneValidationResult Validate()
         {
             return new NzbDroneValidationResult(Validator.Validate(this));
         }
     }
+
+    public enum MetadataLinkType
+    {
+        None = 0,
+        [FieldOption(Label = "IMDb")]
```

Comment:
Should I add `FieldOption` to `None` and `Trakt` as well, where the Enum key aligns with the Label?

Reply:
I think leaving them off as fine as long as the UI is rendering them correctly.